newIDE/app/src/ResourcesList/index.js

```diff
@@ -79,6 +83,47 @@ export default class ResourcesList extends React.Component<Props, State> {
     this.props.onDeleteResource(resource);
   };
 
+  _scanForNewResources = () => {
+    const project = this.props.project;
+    const resourcesManager = project.getResourcesManager();
+    console.log('Scanning the project folder for new resources...');
+    const projectPath = path.dirname(project.getProjectFile());
+
+    const getDirectories = (src, callback) => {
+      glob(src + '/**/*.{png,jpg,jpeg,PNG,JPG,JPEG}', callback);
+    };
+    getDirectories(projectPath, (err, res) => {
+      if (err) {
+        console.log('Error loading ', err);
+      } else {
+        res.forEach(pathFound => {
+          const fileName = path.relative(projectPath, pathFound);
+          if (!resourcesManager.hasResource(fileName)) {
+            const imageResource = new gd.ImageResource();
+            imageResource.setFile(fileName);
+            imageResource.setName(fileName);
+            resourcesManager.addResource(imageResource);
+            imageResource.delete();
+            console.log(fileName + ' added to Project!');
```

Comment:
No need for a capital P nor an exclamation point, and as it's only information, we can use console.info:
```
console.info(`${fileName} added to project.`);
```